modules/virt-automatic-certificates-renewal.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-automatic-certificates.adoc
+
+[id="virt-automatic-certificates-renewal_{context}"]
+= Automatic renewal of TLS certificates
+
+TLS certificates for {VirtProductName} are created during installation for these components:
+
+* KubeVirt
+* containerized data importer controller(CDI)
+* MAC pool
+
+All certificates are renewed and rotated automatically at periodic intervals. You are not required to rotate and refresh TLS certificates manually.
+
+[IMPORTANT]
+====
+When the TLS certificates are being renewed automatically, the following operations continue to function without being cancelled or disconnected:
+
+* Migrations
+* Image uploads
+* VNC and console connections
+====
+
+During the automatic renewal, existing TLS certificates are deleted and replaced with new TLS certificates. The frequency of certificate rotation for each {VirtProductName} component varies.
```

Comment:
I think this module needs to be more minimalistic, and I don't think the other module is necessary. You can put all of the relevant information here in list form.

Tables are best avoided for accessibility reasons unless you're presenting complex data (https://medium.com/@heyoka/lets-bring-table-to-the-table-again-f1ae751159d5).

I also don't think that an [IMPORTANT] note is necessary here. We usually reserve those for serious warnings, and this note is not warning the user of anything. I think this information can also go in a simple list.

I would do something like this (note the corrected CDI capitalization):

```
TLS certificates are automatically deleted and replaced according to the following schedule:
* KubeVirt certificates are renewed daily.
* Containerized Data Importer (CDI) controller certificates are renewed every 15 days.
* MAC pool certificates are renewed every year.

The following operations continue without disruption during automatic renewal:
* Migrations.
* Image uploads.
* VNC and console connections.
```

^ One concern about that last list: are any operations disrupted? Since this is a subset of possible operations, it makes it sound like other operations *might* be disrupted.

Reply:
Table and important note were removed.
Second module was removed.
Copied and replaced existing text with the text shown above.
Checked with Kedar, we don't know if specific operations are impacted. Requested Roman to weigh in as well. Will follow up with Stu and Roman.

Reply:
The approach taken to cert rotation is designed to ensure that there's a valid certificate chain at all times. No impact to *any* interaction with the API is expected. zero downtime. Working a statement such as this into the doc would be appropriate:

```
No operations are disrupted by automatic TLS certificate rotation
```